index.html

```diff
@@ -600,6 +600,13 @@ <h2>Protected Header Parameters</h2>
         <code>alg</code>, <code>iat</code>,
         <code>exp</code> and <code>cnf</code>.
       </p>
+      <p>
+        When <code>iat</code> or <code>exp</code> JWT claims are present,
```

Comment:
When -> When the

Reply:
@selfissued's suggestion plus another
```suggestion
        When the <code>iat</code> and/or <code>exp</code> JWT claims are present,
```